### DISCUSSION TITLE
Aspire CLI tool

### DISCUSSION THREAD

**Discussion by Kralizek** [microsoft/aspire#3214] [Ideas]
it would be cool if we could have a CLI tool to facilitate ordinary tasks like running a project from terminal and so on.

Ideally, it could be very similar to the good old `tye` CLI.

For example, `aspire run` would find the AppHost project in the solution and run it, sparing us from `dotnet --project ./path/to/AppHost run`.

Another command could be `aspire publish` to generate the manifest.

And so on.

**Comment by nickna** [comment]
This idea will make sense when Aspire fully supports GCP and AWS. At that point, a cloud-agnostic deployment tool would rock. It'll probably be up to us, the community, to build it!

**Reply by Kralizek** [reply]
I don't see why support for GCP and AWS (or "a cloud agnostic deployment tool" at all) is important for aspire.

Aspire is a inner cycle development tool. for deployment we have azd for Azure and for k8s we got aspir8)
